Modules/DevTools/TestFramework/TestRunner/src/ALTestTool.Page.al

```diff
@@ -194,7 +194,7 @@ page 130451 "AL Test Tool"
                     PromotedCategory = Process;
                     PromotedIsBig = true;
                     PromotedOnly = true;
-                    ToolTip = 'Runs selected tests.';
+                    ToolTip = 'Runs marked tests.';
```

Comment:
Nice spot. I would drop marked since it is a synonym with selected. So just runs tests or Runs tests from the suite.

Reply:
For an end user with prior Navision / BC experience, selecting and marking has always been considered different actions in the commonly used language ...

That is why I wanted to differentiate between the two actions by using each word in its context.

Reply:
(sorry for my English, it is not so good as I would like)